### PR TITLE
Basic block breaking and placing

### PR DESCRIPTION
**Pull request by ShrimpyStuff**

### CODE REVIEW

File: examples/web_client/index.js [hunk 1]
```diff
@@ -121,6 +122,29 @@ async function main () {
...
+      const ButtonBlock = bot.blockAtCursor(BlockDistance);
+      if (!ButtonBlock) return;
+      if (e.button === 0) {
+        if (bot.canDigBlock(ButtonBlock)) {
+          bot.dig(ButtonBlock);
+        }
+      } else if (e.button === 1) {
+        
+      } else if (e.button === 2) {
+        const Vec3 = require('vec3').Vec3;
```
Comment (extremeheat) [inline examples/web_client/index.js:136]: require this at the top of the file

File: examples/web_client/index.js [hunk 2]
```diff
@@ -121,6 +122,29 @@ async function main () {
...
+    document.addEventListener('mousedown', (e) => {
+      let BlockDistance = (bot.gamemode == 1 ) ? 8 : 5;
+      const ButtonBlock = bot.blockAtCursor(BlockDistance);
+      if (!ButtonBlock) return;
+      if (e.button === 0) {
+        if (bot.canDigBlock(ButtonBlock)) {
+          bot.dig(ButtonBlock);
+        }
+      } else if (e.button === 1) {
+
```
Comment (extremeheat) [inline examples/web_client/index.js:134]: empty condition here

**Review by extremeheat — CHANGES_REQUESTED** [review]
You should lint the file, there's formatting errors

File: examples/web_client/index.js (hunk 2 above)
Comment (ShrimpyStuff) [reply examples/web_client/index.js:134]: Yeah I forgot about that

File: examples/web_client/index.js (hunk 1 above)
Comment (ShrimpyStuff) [reply examples/web_client/index.js:136]: Yeah, I could do that

**Comment by rom1504** [comment]
Any reason to close it ?

**Comment by u9g** [comment]
> Any reason to close it ?

reopened here https://github.com/PrismarineJS/prismarine-viewer/pull/138

**Comment by ShrimpyStuff** [comment]
I closed it because I deleted the repo before I could finish it

**Pull request closed without merging**